event-handler/event_handler_test.go

```diff
@@ -279,9 +279,6 @@ func (s *EventHandlerSuite) TestEvents() {
 	t.Log("STDERR", cmdStderr.String())
 	require.NoError(t, err)
 
-	err = stdinPipe.Close()
```

Comment:
Is this what was failing on Darwin? Or is this what was causing failures on a fancy prompt?

Reply:
This failed when I ran the tests in Darwin using go1.20, with an already closed like error.
It works on linux (at least using go.19)

However, according to docs, we should only close if we want to close it sooner than the program ends.
https://pkg.go.dev/os/exec#Cmd.StdinPipe
> A caller need only call Close to force the pipe to close sooner.